docs/orchestration/agents/kubernetes.md

```diff
@@ -25,7 +25,7 @@ pip install prefect[kubernetes]
 ::: tab Conda
 
 ```bash
-conda install -c conda-forge prefect kubernetes
+conda install -c conda-forge prefect python-kubernetes
```

Comment:
This might need to be something different
```suggestion
conda install -c conda-forge prefect kubernetes python-kubernetes
```

Reply:
My guess is that they would need both the `kubernetes` binary as well as the python client `python-kubernetes`

Reply:
I don't think this is actually required, since we're not running a cluster ourselves.